easybuild/easyconfigs/l/LZO/LZO-2.06-goolf-1.4.10.eb

```diff
@@ -21,10 +21,12 @@ source_urls = [homepage + 'download/']
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 
 sanity_check_paths = {
-    'files': [],
+    'files': ['lib/liblzo2.a', 'lib/liblzo2.so'],
     'dirs': ['lib', 'include']
```

Comment:
Don't really need lib in the dirs check if we explicitly check for files in lib

Reply:
I agree. I only added the new check but I didn't remove the old one. I will do it